shell/pages/c/_cluster/apps/charts/index.vue

```diff
@@ -115,7 +128,10 @@ export default {
         { value: CATALOG_SORT_OPTIONS.LAST_UPDATED_DESC, label: this.t('catalog.charts.sort.lastUpdatedDesc') },
         { value: CATALOG_SORT_OPTIONS.ALPHABETICAL_ASC, label: this.t('catalog.charts.sort.alphaAscending') },
         { value: CATALOG_SORT_OPTIONS.ALPHABETICAL_DESC, label: this.t('catalog.charts.sort.alphaDescending') },
-      ]
+      ],
+      initialVisibleChartsCount: 20,
```

Comment:
Might be worth checking how many charts are initially visible in some high dpi monitors.

I had a look at some of Ovi's figma pages for the initial design and it looks like he had 15 visible (last row was cut off). 

Feels like we could bump this up to 30 to help avoid the multiple scroll movements to reach the bottom